index.js

```diff
@@ -74,10 +74,13 @@ module.exports = ExportNodeModules;
  * @return {String}
  */
 function getLicenses(packageJson) {
-  if (packageJson.licenses) {
+  if (packageJson.licenses && packageJson.licenses instanceof Array) {
     return packageJson.licenses
       .map(license => license.type)
       .join(', ');
+  } else if (packageJson.licenses) {
+    return (packageJson.licenses && packageJson.licenses.type) ||
```

Comment:
Sorry, I missed this. 

Yeah, it looks a bit convoluted but I also have no better idea at the moment. Maybe it's a good idea to add a TODO comment to mark the place for refactoring.